repo/packages/K/koding/0/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "2.0",
+  "name": "koding",
+  "version": "0.0.1",
+  "tags": ["mesosphere", "koding"],
```

Comment:
you can use others like IDE or whatever keyword a user should use to find Koding's package